src/test/java/com/wikia/webdriver/common/driverprovider/NewDriverProvider.java

```diff
@@ -127,7 +123,8 @@ private static EventFiringWebDriver getFFInstance() {
     // to path where executive file of FF is placed
     if ("WINDOWS 8".equals(System.getProperty("os.name").toUpperCase())) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "os.name" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1157)